bin/checkout.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+// node dependencies
+var fs = require('fs');
+
+// external dependencies
+var shell = require('shelljs');
+
+// parse args
+var argv = require('optimist')
+    .usage('Usage: $0 --config [path]')
+    .demand('config')
+    .argv;
+
+function cloneProject(projectName, projectsConfig) {
+
+    var project  = projectsConfig[projectName];
+    var codebase = project.codebases[project.codebase];
+    var command  = 'git clone ' + codebase.repo + ' --branch=' + codebase.branch + ' --depth 1'
+
+    shell.exec(command, {async: false}, function (returnCode, output) {
```

Comment:
Good catch! I forgot to remove the `{async: ...}` bit. Removed in latest commit.